src/lib.rs

```diff
@@ -37,7 +37,7 @@ impl Plugin for AppPlugin {
                 })
                 .set(WindowPlugin {
                     primary_window: Window {
-                        title: "bevy_quickstart".to_string(),
+                        title: "Bevy Quickstart".to_string(),
```

Comment:
Just curious if this might be changing, with the repo moving to `bevy_new_2d`, will we be referring to it as `2D Template` or similar?

Reply:
Ah yeah, we probably will. Can be a separate PR when the rename happens, though.